mlrun/db/httpdb.py

```diff
@@ -1253,6 +1254,8 @@ def list_artifacts(
                 "results", tag="*", project="iris", labels=["uploaded", "type=binary"]
             )
 
+        :param parent: The parent artifact uri. This is used to filter artifacts that
+                        are children of a specific artifact.
```

Comment:
Is it just the URI? I see in the Crud you allow a store URI or just a string which is the `key:tag`. This should be noted.